esp32_s3_korvo_2/esp32_s3_korvo_2.c

```diff
@@ -577,3 +418,27 @@ esp_err_t bsp_led_set(const bsp_led_t led_io, const bool on)
     BSP_ERROR_CHECK_RETURN_ERR(esp_io_expander_set_level(io_expander, led_io, !on));
     return ESP_OK;
 }
+
+esp_err_t bsp_button_init(const bsp_button_t btn)
+{
+    //TODO: initialize IOT buttons and handle it in BSP
+    return ESP_OK;
```

Comment:
Do you have any plans with this TODO?

Reply:
If you remember, we told about it before two weeks. I removed TODO and I will put it into JIRA.

Reply:
Great, thank you!